README.md

```diff
@@ -59,31 +73,57 @@ Each of the directories only include the contents of the zip file that is sent t
 sure to sanitize `aws.env` and `secrets.env` files of all sensitive data, and update their extension to be
 `.env.example`.
 
-### `/data/apis/{oisTitle}/templates`
+### `/data/{protocol/{oisTitle}/templates`
 
 Hosts files containing template data and metadata. The files are named to describe what the template is for so that they
-are human-browseable. The files contents are:
+are human-browsable. The files contents are:
 
 - `airnode`: Airnode address (must match the one in `apiMetadata.json`)
 - `endpointId`: Endpoint ID (must match one from `config.json`)
 - `parameters`: Airnode ABI-encoded parameters
 - `templateId`: Template ID (must match `airnode`, `endpointId`, `parameters` encoded and hashed)
 - `decodedParameters`: `parameters` decoded to be human-readable
-- `chains`: Chains that the template is currently deployed on (must match ones from `chains.json`)
+- `versionedChains`: Chains that the template is currently deployed on (must match ones from `chains.json`), with
+  deployment versioning.
+
+---
+
+### `/data/proto-psp/{oisTitle}/beacons`
+
+Files containing subscription data and metadata. The files are named to describe what the subscription is for so that
+they are human-browsable.
+
+- `beaconId`: A subscription ID, eg. "0x168194af62ab1b621eff3be1df9646f198dcef36f9eace0474fd19d47b2e0039"
+- `name`: A descriptive name for human reference, eg. "Coingecko USD/BTC"
+- `description`: An extended description of the subscription, eg. "Price of BTC in USD"
+- `airnodeAddress`: "0xA30CA71Ba54E83127214D3271aEA8F5D6bD4Dace"
+- `templateId`: An associated template ID, eg. "0xea30f92923ece1a97af69d450a8418db31be5a26a886540a13c09c739ba8eaaa"
+- `parameters`: Override parameters"0x"
+- `conditions`: ABI Encoded condition inputs, eg. "0x316242..."
+- `conditionsDescription`: A human-readable description of the `conditions` configuration directive
+- `relayer`: The relayer address, eg. "0xA30CA71Ba54E83127214D3271aEA8F5D6bD4Dace"
+- `sponsor`: The sponsor address, eg. "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC"
+- `requester`: The requester address, eg. "0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0"
+- `fulfillFunctionId`: "0x206b48f4",
+- `chains`: [...] Chains that the beacon is currently operational on
+  - `namedVersion`: Name of the chain (must match one from `chains.json`) (from `chains.json`, eg. `["ropsten", "v1"]`)
+  - `topUpWallets`: [...]
+
+---
 
-### `/data/apis/{oisTitle}/beacons`
+### `/data/rrp-psp/{oisTitle}/beacons`
 
-Hosts files containing beacon data and metadata. The files are named to describe what the beacon is for so that they are
+Files containing beacon data and metadata. The files are named to describe what the beacon is for so that they are
 human-browsable.
 
 - `templateId`: Template ID (must match one from `/data/apis/{oisTitle}/templates`)
-- `templateName`: A human-readable name for the template, suitable for documentation use.
```

Comment:
Named as such to avoid confusion with the beacon name (which is the file name)

Reply:
Very useful, thanks, I was wondering.

Reply:
This has been added back even though it isn't indicated as such here.